app/src/main/java/com/example/kindergarten_management/helpers/SqlDatabaseHelper.java

```diff
@@ -133,8 +134,8 @@ public StaffMemberModel readStaffMember(int staffMemberId) {
             staffMember.setName(cursor.getString(cursor.getColumnIndexOrThrow(STAFF_COLUMN_NAME)));
             staffMember.setRule(cursor.getString(cursor.getColumnIndexOrThrow(STAFF_COLUMN_RULE)));
```

Comment:
In this code:
   StaffMemberModel staffMember = new **StaffMemberModel(getContext());**
            staffMember.setId(cursor.getInt(cursor.getColumnIndexOrThrow(STAFF_COLUMN_ID)));
            staffMember.setName(cursor.getString(cursor.getColumnIndexOrThrow(STAFF_COLUMN_NAME)));
            staffMember.setRule(cursor.getString(cursor.getColumnIndexOrThrow(STAFF_COLUMN_RULE)));

Please don't use the constructor with the content.
Use the constructor the get the parameters